docs/translation/index.md

```diff
@@ -99,7 +99,21 @@ The general rules for avoiding errors may be summed up like this:
     ```smarty
     <p>{ts}Hello, world!{/ts}</p>
     ```
-    
+
+An exception is `<a>` links which should be within the `ts`, but its parameters should be arguments to the ts. For example:
+
+!!! failure "Bad"
+
+    ```smarty
+    {ts}<a href="https://www.civicrm.org" target="_blank">CiviCRM Web Site</a>{/ts}
+    ```
+
+!!! success "Less bad"
+
+    ```smarty
+    {ts 1='href="https://www.civicrm.org" target="_blank"'}<a %1>CiviCRM Web Site</a>{/ts}
+    ```
+
```

Comment:
What do you think to a "multi-ts" example here?

The scenario is this:

I have a link - I need to translate the link text with `ts`. That link has title text which I also want to pass through `ts` - how can this be done "properly".

I'm just thinking of what examples I've needed to lookup in the past here.

Reply:
I'm not sure if it's github's display or something wonky but the updated text clarifies (or tries to clarify) that there's a difference between links in larger blocks of text vs standalone links. The former rarely have title attributes because the block of text _is_ the text.